src/subscription/__factories__/subscription.factory.js

```diff
@@ -0,0 +1,56 @@
+import { Factory } from 'rosie'; // eslint-disable-line import/no-extraneous-dependencies
+// import Chance from 'chance'; // eslint-disable-line import/no-extraneous-dependencies
+
+import '../../payment/__factories__/product.factory';
+import '../../payment/__factories__/message.factory';
+
+// const chance = new Chance(12345);
+
+// const getDueDate = () => {
+//   const dueDate = new Date();
+//   dueDate.setDate(dueDate.getDate() + 8); // after 7 days free trial period
+
+//   const formattedDate = dueDate.toLocaleDateString(
+//     {},
+//     {
+//       timeZone: 'UTC',
+//       month: 'long',
+//       day: '2-digit',
+//       year: 'numeric',
+//     },
+//   );
+//   const d = formattedDate.split(' ');
+//   return `${d[0]} ${d[1]} ${d[2]}`;
+// };
+
+Factory.define('subscription')
+  .sequence('subscription_id')
+  .option('numProducts', 0)
+  .option('productType', 'Course Entitlement')
+  .option('numErrorMessages', 0)
+  .option('numInfoMessages', 0)
+  .attrs({
+    program_title: 'Blockchain Fundamentals',
+    organization: 'University of California, Berkeley',
+    certificate_type: 'verified',
+    // due_date: getDueDate(),
```

Comment:
Let's not forget to discuss this in standup today!

Reply:
Update here is that we'll get this date from the `subscription` BE, not directly from Stripe from the MFE. BE will get it from Stripe.

Reply:
I think we created a ticket for it to send it from BE?

Reply:
ah! you already ready replied, thanks 
> Update here is that we'll get this date from the subscription BE, not directly from Stripe from the MFE. BE will get it from Stripe.